DataCollection/src/Migrations/201602271714081_SurveyTypeWindSpeedRename.cs

```diff
@@ -0,0 +1,24 @@
+namespace FlightNode.DataCollection.Domain.Migrations
```

Comment:
Please go ahead and delete this file from Visual Studio, so that we don't have conflicting migrations. If you have already installed this in your database, then you will need to roll back to the previous migration.